test/cypress/integration/card/billing-address-collection.cy.test.js

```diff
@@ -78,12 +80,9 @@ describe('Billing address collection', () => {
     const createPaymentChargeStubs = cardPaymentStubs.buildCreatePaymentChargeStubs(
       tokenId, chargeId, 'en', gatewayAccountId, serviceOpts, {}, {}, chargeOpts)
 
-    beforeEach(() => {
-      // preserve cookies so we can prodeed with payment flow
-      Cypress.Cookies.preserveOnce('frontend_state')
-    })
+    it('Should not show the billing address section and allow valid card submission', () => {
+      cy.log('Should not show the billing address section')
```

Comment:
Any reason we need this log which is similar to the test name?

Reply:
Updated this as per above.